pos_accounting/src/pool/delta/mod.rs

```diff
@@ -47,6 +47,14 @@ impl<'a> PoSAccountingDelta<'a> {
         &self.data
     }
 
+    pub fn is_empty(&self) -> bool {
```

Comment:
Typically methods where you do the same thing for all the members are a fire hazard. The reason is that if a member were to be added in the future, you'll have to track it yourself and not forget to do it. This is why rust has the Eq, Ord derive and similar. It solves this problem. I'm OK with this as a temporary solution, but please throw in some TODO to make this some kind of derive in the future to avoid manually specifying members.

Also, it makes more sense to implement this for `PoSAccountingDeltaData` then just call it here.